pages/dkp/kommander/2.2/legal/version-policy/index.md

```diff
@@ -64,6 +64,13 @@ Support for Experimental components is provided on a Standard level, Severity 4
 
 This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.
 
+### Technical preview
+
+Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:
```

Comment:
I think it's exactly the same as Experimental, and we should either copy pasta that section here, or add the words technical preview to the 'Experimental' section header.